clients/drcachesim/tests/drmemtrace.threadsig.aarch64.raw/README

```diff
@@ -0,0 +1,50 @@
+How to re-generate the drmemtrace.threadsig.aarch64.raw files
+
+The application run is "threadsig" from https://github.com/DynamoRIO/drmemtrace_samples/blob/master/threadsig.aarch64/threadsig.cp
+
+To save repository space, build it with -Os and -static on an AArch64
+machine or cross-compile it:
+
+  $ g++ -o threadsig -Os threadsig.cpp -lpthread -static
+  $ strip threadsig
+
+Now run it with 4 threads and 2000 iterations, and possibly
+-unsafe_build_ldstex depending on your architecture:
+
+  $ bin64/drrun -unsafe_build_ldstex -t drcachesim -offline -- ~/threadsig 4 2000
+
+Compress the raw files:
+
+  $ gzip `ls -1td drmemtrace.threadsig.*.dir|head -1`/raw/*.raw
+
+Now copy the new data files and the threadsig and libdynamorio.so.
+
+Strip libdynamorio.so, though this may not decrease it much:
+
+  $ strip lib64/release/libdynamorio.so
+
+There is no need to change the checked-in libdrmemtrace.so.  It is not
+executed.  I created it by just copying a pure-asm tiny ELF binary:
+e.g., suite/tests/bin/common.allasm_aarch64_isa.
+
+A smaller libdynamorio.so I created by compiling this file
```

Comment:
The part below seems optional. Perhaps, mark it as such?

Reply:
It does save 300K so I was waffling whether to push it.  WDYT?

Reply:
Probably it's too fiddly and complex to synthesize (have to go find dynamorio_sigreturn address, etc.) or make a tool; will mark optional and we will live with the 300K.